elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/AsyncProperties.java

```diff
@@ -33,6 +33,12 @@
      */
     private int queryCleanupDays = 7;
 
+    /**
+     * Default time interval for cancelling async query transactions
```

Comment:
The other comment for standalone was:
"Number of seconds between async query cancellation checks."

Let's use the same comment for both standalone and spring and maybe change to:
"Polling interval to identify async queries that should be canceled."

Also, 300 seconds is way too long for this interval.  It should probably be somewhere in the 1-10 second range by default.